### PR TITLE
[full-ci] chore: prepare release v10.0.0

### PR DESCRIPTION
**Pull request by kulmann**

### CODE REVIEW

File: changelog/10.0.0_2024-07-29/change-remove-application-menu [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Change: Remove applicationMenu app property
+
+BREAKING CHANGE for developers: We've removed the `applicationMenu` app property in favour of the new `appMenuItem` extension type.
```
Comment (JammingBen) [inline changelog/10.0.0_2024-07-29/change-remove-application-menu:3]: We just deprecated the prop, does that mean we can actually remove it now? 🤔

**Review by JammingBen — APPROVED** [review]
👍 

We should probably release a `10.0.0` of `web-client` and `web-pkg` as well now, no?

File: changelog/10.0.0_2024-07-29/change-remove-application-menu (hunk 1 above)
Comment (kulmann) [reply changelog/10.0.0_2024-07-29/change-remove-application-menu:3]: You removed all the handling in the runtime 🤔

Comment (JammingBen) [reply changelog/10.0.0_2024-07-29/change-remove-application-menu:3]: No I didn't 😅   I just ported our apps, but the fallback is still active: https://github.com/owncloud/web/blob/renovate/npm-run-all-replacement/packages/web-runtime/src/components/Topbar/TopBar.vue#L186

Comment (kulmann) [reply changelog/10.0.0_2024-07-29/change-remove-application-menu]: ah... hm. sorry 😅 removed the changelog item again.

**Pull request merged**